gen.py

```diff
@@ -23,33 +26,55 @@ def __str__(self: "Visibility") -> str:
         return self.value
 
 
+def get_log_level(no_log: bool, verbose: bool) -> int:
+    if no_log:
+        return logging.ERROR
+    elif verbose:
+        return logging.DEBUG
+    else:
+        return logging.INFO
+
+
 def count_combinations() -> None:
     """Calculate the number of possible outputs"""
     num_folx = len(FOLX)
     num_treats = len(TREATS)
     combinations = num_folx * num_treats
-    print(
-        f"There are {num_folx} folx and {num_treats} treats, resulting in {combinations:,} possible combinations."
-    )
+    output = f"There are {num_folx} folx and {num_treats} treats, resulting in {combinations:,} possible combinations."
+    log.info(output)
+    print(output)
 
 
 def write_status(
     status: str, dry_run: bool = False, visibility: Visibility = Visibility("unlisted")
 ) -> None:
     """Write a status to Mastodon"""
-    mastodon = Mastodon(access_token=config.ACCESS_TOKEN, api_base_url=config.API_URL)
     if dry_run is False:
         # Post
+        mastodon = Mastodon(
+            access_token=config.ACCESS_TOKEN, api_base_url=config.API_URL
+        )
```

Comment:
Moving this in here prevents the script crashing during a dry run if the config isn't set, which it doesn't need for a dry run